deploy/helm/charts/templates/csi-controller.yaml

```
@@ -29,7 +29,7 @@ spec:
           resources:
 {{ toYaml .Values.csiController.resources | indent 12 }}
           args:
-            - "--v=5"
+            - "--v={{ .Values.csiController.logLevel | default .Values.csiController.resizer.logLevel }}"
```

IMO, it should be in below way... 
- If the configuration is not specified then high-level setting will be applied but if it is specified at container level then those settings should be applied
```suggestion
            - "--v={{  .Values.csiController.resizer.logLevel | default .Values.csiController.logLevel }}"
```

Simillarly, In other places and also open for discussion

---

Agreed, but see [my comment on your other suggestion](https://github.com/openebs/cstor-operators/pull/397#discussion_r736404390)